src/redux.js

```diff
@@ -487,13 +487,17 @@ export default function rootReducer(state = initialState, action) {
     };
   }
   if (action.type === SET_HIGHLIGHT_COLUMN) {
-    return {
-      ...state,
-      highlightColumn: action.highlightColumn
-    };
+    if (getShowColumnClicking(state)) {
+      return {
+        ...state,
+        highlightColumn: action.highlightColumn
+      };
+    }
   }
   if (action.type === SET_CURRENT_COLUMN) {
-    if (state.currentColumn === action.currentColumn) {
+    if (!getShowColumnClicking(state)) {
+      // Do nothing.
```

Comment:
Do we need to `return {...state}` here?

Reply:
You're right that this was a bit confusing.  I was trying to have all no-ops fall through to the `return state` at the end of the function, but the logic was getting hard to read.  I'm going to rearrange.

Reply:
Thanks for clarifying & for the comments!